Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -119,6 +119,33 @@ of removing unused segments after release of internal resources.
   > in-progress ArangoDB transactions will still continue to return a
   > repeatable-read state.
 
+  > ArangoSearch performs operations in its index based on numerous writer
+  > objects that mapped to processed segments. In order to control memory that
+  > is used by these writers (in terms of "writers pool") one can use
+  > `writebuffer*` properties of a view.
+
+- **writebufferIdle** (_optional_; type: `integer`; default: `64`;
+  to disable use: `0`)
+
+  Maximum number of writers (segments) cached in the pool. `0` value turns off
```

Comment:
It already says `to disable use: 0` above, can we therefore remove the sentence that repeats this here?